src/redux/categories/categories.js

```diff
@@ -1,15 +1,13 @@
 const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] In keeping with the duck's pattern, kindly re-assign your action type here.

  ```
  const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'
  ```

Reply:
Thank you @julie-ify , I have made the correction